src/main/webapp/EditHelp.js

```diff
@@ -0,0 +1,107 @@
+// Replacement of the function on help-button click event
+Behaviour.specify("A.help-button", 'hudson-behavior', 1, function(e) {
+    e.onclick = function() {
+        var tr = findFollowingTR(this, "help-area");            
+        var thistr = $(this).up().up();
+
+        // Check if the custom help area is open
+        if (!$(tr).hasClassName("custom-help")) {
+            // Created element tr for custom help
+            var customtr = document.createElement("tr");
+            customtr.innerHTML = '  <td></td><td colspan="2"><div class="help"><div class="custom-help-button">'+
+            '<div class="custom-help-button-save" onclick="save_custom_help(this)" >Save</div><div class="custom-help-button-edit" onclick="edit_custom_help(this)" >Edit</div>'+
+            '</div><div class="custom-help-text">Loading1...</div></div></td><td></td>';
+            $(customtr).addClassName("help-area");
+            $(customtr).addClassName("custom-help");
+            $(thistr).insert({after:customtr});
+            // Modification of the inner structure of the created element
+            $(customtr).down().next().down().down().down().next().style.display = "block";
+            // Location for the default help
+            var div = $(tr).down().next().down();
```

Comment:
@michaelneale @lanwen Is the proposed selection possible in prototype js without jQuery?